public/js/init_toast.js

```diff
@@ -0,0 +1,25 @@
+jQuery(() => {
+    window.toast = (c) => window.reactToastify.toast(window.React.createElement("div", { dangerouslySetInnerHTML: { __html: `<h2>${c.heading}</h2><p>${c.text}</p>` } }), {
+        toastId: c.toastId,
+        type: c.icon || c.type || "info",
+        position: c.position || "top-left",
+        onOpen: c.onOpen,
+        onClose: c.onClose,
+        autoClose: c.hideAfter || c.autoClose || ((c.icon || c.type) === "info" || (c.icon || c.type) === "success") ? 5000 : 7000,
+        closeButton: c.allowToastClose ?? c.closeButton ?? true,
+        hideProgressBar: (typeof (c.loader) === "boolean" && !c.loader) ?? c.hideProgressBar ?? false,
+        pauseOnHover: c.pauseOnHover ?? true,
+        pauseOnFocusLoss: c.pauseOnFocusLoss ?? true,
+        closeOnClick: c.closeOnClick ?? ((c.icon || c.type) === "info" || (c.icon || c.type) === "success") ?? false,
+        draggable: c.draggable ?? ((c.icon || c.type) === "info" || (c.icon || c.type) === "success") ?? false,
+    });
+    const toastDiv = document.createElement("div");
+    document.body.appendChild(toastDiv);
+    toastDiv.style.textAlign = "initial";
+    window.React.render(
+        window.React.createElement(window.reactToastify.ToastContainer, {
+            style: {},
+            limit: 7,
+            theme: "colored",
```

Comment:
I believe this is a good opportunity to style the toasts according to the loaded css theme (especially since toastify has colored loading bars and icons depending on the status which look quite nice)
```suggestion
            theme: "light",
```  

We should then define custom `--toastify-color-light` and `--toastify-text-color-light` variables in the theme files. An example for modern.css:  

```
:root {
  --toastify-text-color-light: #DDDDDD;
  --toastify-color-light: #4F535B;
}

.Toastify__close-button--light {
  color: #fff;
  opacity: 1;
}
```  

![image](https://user-images.githubusercontent.com/8237712/166218858-57a49eeb-aad2-48df-b946-b47040e5e6ee.png)
![image](https://user-images.githubusercontent.com/8237712/166219295-e86801be-8640-4005-9356-634240980c0d.png)

Reply:
Yep, I may not cover all the situations. But changing colored to light may result in significant differences. More tests should be done later.

Reply:
I personally prefer the modded-light look to the colored one, but we can adjust that later indeed.